.buildkite/pipeline.yml

```diff
@@ -164,6 +164,16 @@
     - bazel test --config=ci $(./scripts/bazel_export_options)
       --test_tag_filters=-kubernetes,-jenkins_only,medium_size_python_tests_k_to_z,-flaky
       python/ray/tests/...
+- label: ":python: (Worker In Container)"
+  conditions: ["RAY_CI_PYTHON_AFFECTED"]
+  commands:
+    - LINUX_WHEELS=1 ./ci/travis/ci.sh build
+    - pip install docker
+    - python ./ci/travis/build-docker-images.py --py-versions PY36 --build-type BUILDKITE --only-build-nest-container
+    - mkdir /ray-mount/containers
+    - docker save -o /ray-mount/containers/images.tar rayproject/ray-nest-container:nightly-py36-cpu
```

Comment:
please add some comments why are we doing this. i.e. what does each container do, why do we run the test in the container (i assume it needs access to podman?)